.github/workflows/make_bundle_conda.yml

```diff
@@ -4,10 +4,10 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - 'build_installers.py'
-      - 'conda-recipe/*'
-      - '.github/workflows/make_bundle_conda.yml'  # this file
+    # paths:
+    #   - 'build_installers.py'
+    #   - 'conda-recipe/*'
+    #   - '.github/workflows/make_bundle_conda.yml'  # this file
```

Comment:
Why comment out this?

Reply:
Needed to trigger the workflow at first for a control check with `bundle_tools_2`. Thanks for the reminder!

Reply:
```suggestion
    paths:
      - 'build_installers.py'
      - 'conda-recipe/*'
      - '.github/workflows/make_bundle_conda.yml'  # this file
```